awsaws-ses/aws-ses/src/main/java/com/brother/awsses/controller/EmailController.java

```diff
@@ -0,0 +1,24 @@
+package com.brother.awsses.controller;
+
+
+import com.brother.awsses.dto.ReceiverInfo;
+import com.brother.awsses.service.AmazonSesService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("aws-ses-management")
```

Comment:
@RequestMapping("aws-ses-management")
@PostMapping("sendEmail")
Different naming conventions were found. snake case and camelCase.

Reply:
> @RequestMapping("aws-ses-management") @PostMapping("sendEmail") Different naming conventions were found. snake case and camelCase.

change the end-point from sendEmail to send-email